app.R

```diff
@@ -358,7 +362,19 @@ server <- function(input, output, session) {
   # Render the race results table
   output$race_results_table <- DT::renderDT({
     
-    datatable(filtered_race_results())
+    datatable(filtered_race_results(),
+              options = list("pageLength" = 20,
+                             "paging" = FALSE,
+                             "scrollY" = '800px',
+                             "autoWidth" = TRUE
+              ),
+              selection = "none"
+    ) |> 
+      formatStyle(
+        'flag', target = 'row', 
+        backgroundColor = styleEqual(c(1), c('green'))
+      )
+    
```

Comment:
Can you make the highlight this colour? "#B138DD" This is the colour used in the official F1 broadcast. If possible just highlight the fastest lap column this colour!

Reply:
<img width="503" alt="image" src="https://user-images.githubusercontent.com/50146522/222875300-3c5b9ba2-41d9-4d51-89b4-ef20e25fb3f4.png">

Reply:
This fine?

Reply:
Ok, pushing

Reply:
Can you make it just highlight the fastest lap column?